Dockerfile

```diff
@@ -1,13 +1,20 @@
 # Build on top of a Node.js + npm image.
-FROM node:latest
+FROM node:10
 
 # Make a codebase directory and work in it
 RUN mkdir codebase
 WORKDIR /codebase
 
 # Load the package.json and package-lock.json files
-COPY package.json .
-COPY package-lock.json .
+COPY package*.json ./
 
 # Install packages
 RUN npm install
+RUN npm install -g @vue/cli
+
+# Copy over all project files to container
+COPY . ./
```

Comment:
This is not required as we're using Docker Compose to map the project files directly into a volume on the container.